src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static class Node<K, V> {
+        private K key;
+        private V value;
+        private int hash;
+        private Node<K, V> next;
+
+        public Node(K key, V value, int hash, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.hash = hash;
+            this.next = next;
+        }
+    }
+
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int COEFFICIENT_GROW = 2;
+    private Node<K, V>[] map;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        capacity = 16;
+        map = (Node<K, V>[]) new Node[capacity];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        putItem(map, capacity, key, value);
+        if (size > capacity * LOAD_FACTOR) {
+            resizeMap();
+        }
```

Comment:
maybe here better to swap putItem() with if() , couse if size already >= threshould , first what we should to do, its resize our map

Reply:
ok